plugins/discourse-reactions/assets/javascripts/discourse/components/discourse-reactions-counter.gjs

```diff
@@ -203,12 +204,12 @@ export default class DiscourseReactionsCounter extends Component {
 
         {{#if (and @post.yours this.onlyOneMainReaction)}}
           <div class="discourse-reactions-reaction-button my-likes">
-            <button
+            <DButton
```

Comment:
If this is converted to a `DButton`, you can remove `type="button"` and replace the block contents with `@icon={{this.siteSettings.discourse_reactions_like_icon}}`